eventarc/generic/README.md

```diff
@@ -0,0 +1,49 @@
+# Cloud Eventarc - Generic tutorial
+
+This sample shows how to create a service that processes generic CloudEvents.
```

Comment:
Users may confused what a CloudEvent is here. Maybe add a link or more explanation.

Reply:
Added a link.

In general, we're trying to reduce the amount of duplicate instructions between CloudSite and the docs, so we might remove this README in the future like we do with `functions/`.